node/node.go

```diff
@@ -118,7 +118,12 @@ func New(conf *Config) (*Node, error) {
 				rotateHours = *conf.LogConfig.RotateHours
 			}
 
-			log.Root().SetHandler(log.NewFileLvlHandler(logFilePath, *conf.LogConfig.MaxBytesSize, *conf.LogConfig.Level, rotateHours))
+			maxBackups := uint(0)
```

Comment:
fistly
1. maxBackups means number of backups, not include bsc.log self
2. unlike rotateHours, 0 here is a meaningful value, so default it to -1, and check conf.LogConfig.MaxBackups>=0